### PR TITLE
Cloudino Guide

### PR DESCRIPTION
**Pull request by samjm**
Hello, this is the completed cloudino guide following the initial structure. Best Regards.

### CODE REVIEW

File: docs/sensors/cloudino.md [hunk 1]
```diff
@@ -1,9 +1,392 @@
...
+For the purpose of this guide we will use the following components to develop our
+air quality sensor unit:
+
+* Cloudino WiFi Connector (CWC)
+* Arduino UNO
+* PPD42NS. PM10 Sensor
+* DHT11. Temperature / Humidity Sensor
+* MQ-131. Ozone Sensor
+* Grove Multichannel. Gas Sensor (NO2, CO)
+
```
Comment (chicco785) [inline docs/sensors/cloudino.md]: can you provide few references where you can acquire the needed components?
especially for cloudino (i suppose it is easy for people to google where to order and arduino, but how do you procure a cloudino?)

**Review by chicco785 — CHANGES_REQUESTED** [review]

File: docs/sensors/cloudino.md [hunk 2]
```diff
@@ -1,9 +1,392 @@
...
+  *Fig. 8. WiFi network generated by any CWC*
+
+For instance, Fig. 8 highligths Cloudino_FAAE37 network that is used to
+configure the particular CWC (Cloudino_FAAE37 device).
+
+As soon as the WiFi connection is made, a configuration panel shows up. In case
+it doesn't happen, you can access to the panel using a web browser with the
+**192.168.4.1** IP address. Fig. 9 shows the main configuration Web page for a CWC.
+
+  ![CWC Configuration Panel on 192.168.4.1](./images//cdn_09.png)
```
Comment (chicco785) [inline docs/sensors/cloudino.md:137]: is this a fixed IP on cloudino right? so it's always the same ip?

Comment (samjm) [reply docs/sensors/cloudino.md:137]: Yes, this is a fixed local IP from Cloudino and is always the same. It usually access directly once Cloudino is connected

File: docs/sensors/cloudino.md (hunk 1 above)
Comment (samjm) [reply docs/sensors/cloudino.md]: It has been updated. Let me know if it's ok, thanks!

**Review by chicco785 — APPROVED** [review]

**Pull request merged**